Felienne/hedy
# [FIX] #1468

**Description**
Currently we set a `min-height` attribute on the `adventures-tab`. When switching tabs we determine if it is either an adventure (set height to `20em`) or the quiz (set height to `100%`). The issue with the current implementation is that due to the `min-height` attribute the first tab would have infinite height and only gain the `20em` limit after switching tabs. In this PR we fix this issue by setting a fixed height of `20em` and only removing it when reaching the quiz tab.

**Fix for**
This PR fixed #1468.

**How to test**
Navigate to level 5 (which has a large text for the first tab) and notice that the max-height is set correctly. In contrast with the screenshots in the issue #1468.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
